server/src/test/java/org/elasticsearch/index/codec/PerFieldMapperCodecTests.java

```diff
@@ -49,6 +49,100 @@ public void testUseBloomFilterWithTimestampFieldEnabled_disableBloomFilter() thr
         );
     }
 
+    public void testUseES87TSDBEncodingForTimestampField() throws IOException {
```

Comment:
Maybe also add a test that uses `index.time_series.es87tsdb_codec.enabled` setting